joomla/joomla-cms
# [4.1] Update fix 4.1.0-2021-11-20.sql

Pull Request for Issue # .

### Summary of Changes

Fix empty parameter for mail template.

### Testing Instructions

1. Make a new installation of 4.1 Beta 3 and check in database the parameter modified by this PR here.
Result: The parameter is wrong `'{"tags": ["task_id", "task_title", ""]}'`.
2. Update the 4.1 beta 3 from step 1 to this PR by using either the patched package https://ci.joomla.org/artifacts/joomla/joomla-cms/4.1-dev/36576/downloads/50275/Joomla_4.1.0-dev+pr.36576-Development-Update_Package.zip or the custom update URL https://ci.joomla.org/artifacts/joomla/joomla-cms/4.1-dev/36576/downloads/50275/pr_list.xml , check if it works without SQL errors and check in database if the empty parameter changed by this PR has been fixed.
Result: The parameter is right `'{"tags": ["task_id", "task_title"]}'`.

### Documentation Changes Required

None.
